n3fit/src/n3fit/backends/keras_backend/operations.py

```diff
@@ -290,6 +290,21 @@ def pow(tensor, power):
     return tf.pow(tensor, power)
 
 
+@tf.function
+def absolute(tensor):
+    """
+    Compute the absolute value of a tensor
+    """
+    return K.abs(tensor)
+
+
+def multiply_minusone(tensor):
```

Comment:
Thanks @scarlehoff for the comments! I will first try to reply to your questions and then will address the rest in some updated commits.

> You are already implementing a new operation, you can have the -abs() there if you really need to.

That was indeed what I tried at the very beginning but then got some errors because of type errors. But I agree that it will be great to have something like this and remove these functions. I will have another look again and try to fix the issues (maybe things have changed since).

Reply:
Things have changed indeed and now this works!! Not only this removes the unneeded functions but also simplifies the `observable` a lot.